BUILDING.md

```diff
@@ -113,8 +113,8 @@ platforms. This is true regardless of entries in the table below.
 | Windows          | x86 (native)     | >= Windows 8.1/2012 R2          | Tier 1 (running) / Experimental (compiling) <sup>[6](#fn6)</sup> | |
 | Windows          | x64, x86         | Windows Server 2012 (not R2)    | Experimental |                                   |
 | Windows          | arm64            | >= Windows 10                   | Tier 2 (compiling) / Experimental (running) |    |
-| macOS            | x64              | >= 10.13                        | Tier 1       |                                   |
-| macOS            | arm64            | >= 11                           | Experimental |                                   |
+| macOS            | x64              | >= 10.13                        | Tier 1       | Our Binaries are compiled with 10.13 as a target there is no guarrentee compiling on 10.13 will work as Xcode11 is required to compile. |
```

Comment:
```suggestion
| macOS            | x64              | >= 10.13                        | Tier 1       | Our Binaries are compiled with 10.13 as a target there is no guarantee compiling on 10.13 will work as Xcode11 is required to compile. |
```
Although given the length of the sentence you may want to turn it into a note and add below the table.

Reply:
done.